install/kots/manifests/gitpod-installer-job.yaml

```diff
@@ -303,6 +303,11 @@ spec:
                 'del(select(.kind == "StatefulSet" and .metadata.name == "openvsx-proxy").status)' \
                 "${GITPOD_OBJECTS}/templates/gitpod.yaml"
 
+              echo "Gitpod: Update GITPOD_INSTALLATION_PLATFORM telemetry value"
+              yq eval-all --inplace \
+                '(select(.kind == "CronJob" and .metadata.name == "gitpod-telemetry") | (.spec.jobTemplate.spec.template.spec.containers[0].env[] | select(.name=="GITPOD_INSTALLATION_PLATFORM")).value |= "repl{{ Distribution }}"' \
+                "${GITPOD_OBJECTS}/templates/gitpod.yaml"
```

Comment:
Could we maybe find a cleaner solution than this `yq` magic? This makes the installer job unnecessarily complicated, IMHO. Maybe a proper config value for the installer config?

Reply:
I guess we could stick it in the `experimental` config as we probably don't want to make this a publicly configurable parameter and default to `installer`. Only question is whether the Local Preview runs it in experimental mode

Reply:
We can update the `local-preview` to also run in experinmental mode. So, Wokring on adding this into the config